pkgs/by-name/me/meteo-qt/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  qt5,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "meteo-qt";
+  version = "3.4";
+
+  src = fetchFromGitHub {
+    owner = "dglent";
+    repo = "meteo-qt";
+    rev = "v${version}";
+    hash = "sha256-J9R6UGfj3vaPfn0vmjeRMsHryc/1pxoKyIE9wteVYbY=";
+  };
+
+  nativeBuildInputs =
+    with python3Packages;
+    [
+      sip
+    ]
+    ++ [
+      qt5.qttools
+      qt5.wrapQtAppsHook
+    ];
+
+  propagatedBuildInputs = with python3Packages; [
+    lxml
+    pyqt5
+  ];
+
+  pythonImportsCheck = [ "meteo_qt" ];
+
+  makeWrapperArgs = [ "\${qtWrapperArgs[@]}" ];
+
+  meta = with lib; {
+    description = "System tray application for weather status information";
+    homepage = "https://github.com/dglent/meteo-qt";
+    changelog = "https://github.com/dglent/meteo-qt/blob/${src.rev}/CHANGELOG";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ linuxissuper ];
```

Comment:
```suggestion
  meta = {
    description = "System tray application for weather status information";
    homepage = "https://github.com/dglent/meteo-qt";
    changelog = "https://github.com/dglent/meteo-qt/blob/${src.rev}/CHANGELOG";
    license = lib.licenses.gpl3Only;
    maintainers = with lib.maintainers; [ linuxissuper ];
```

Reply:
I don't have a problem with this suggestion but why is it better?